airflow/contrib/example_dags/example_gcf_function_deploy_invoke_delete.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that displays interactions with Google Cloud Functions.
+It creates a function, invokes and then deletes it.
+
+This DAG relies on the following Airflow variables
+https://airflow.apache.org/concepts.html#variables
+* PROJECT_ID - Google Cloud Project to use for the Cloud Function.
+* REGION - Google Cloud Functions region where the function should be
+  created.
+* ENTRYPOINT - Name of the executable function in the source code.
+* and one of the below:
+    - SOURCE_ARCHIVE_URL - Path to the zipped source in Google Cloud Storage
+    or
+    (
+        - SOURCE_UPLOAD_URL - Generated upload URL for the zipped source
+        and
+        - ZIP_PATH - Local path to the zipped source archive
+    )
+    or
+    - SOURCE_REPOSITORY - The URL pointing to the hosted repository where the function is
+    defined in a supported Cloud Source Repository URL format
+    https://cloud.google.com/functions/docs/reference/rest/v1/projects.locations.functions#SourceRepository
+"""
+
+import datetime
+
+import airflow
+from airflow import models
+from airflow.contrib.operators.gcf_function_delete_operator import GCFFunctionDeleteOperator  # nopep8
```

Comment:
Please remove nopep8 and make lines < 90 chars (as per Airflow's fluke8 automated tests)

Reply:
Done